tests/test_demos.py

```diff
@@ -7,8 +8,19 @@
 skip_no_plotting = pytest.mark.skipif(not system_supports_plotting(),
                                       reason="Test requires system to support plotting")
 
+# this will have to be modified once VTK finalizes how they release
+# dev wheels
+try:
+    from vtkmodules.vtkCommonCore import vtkVersion
+    vtk_dev = len(str(vtkVersion().GetVTKBuildVersion())) > 2
+except:
+    vtk_dev = False
 
 
+# These tests fail with mesa opengl on windows
+skip_windows_dev_whl = pytest.mark.skipif(os.name == 'nt' and vtk_dev,
+                                          reason='Test fails on Windows with dev wheels')
```

Comment:
Just to match the other one if that suggestion is accepted
```suggestion
                                          reason='Test fails on Windows with VTK dev wheels')
```